server/routes/utils/adHelpers.ts

```diff
@@ -436,3 +526,59 @@ export const getLatestDetectorTasksQuery = () => {
 export const isRealTimeTask = (task: any) => {
   return get(task, 'task_type', '').includes(REALTIME_TASK_TYPE_PREFIX);
 };
+
+export const getLatestTasksForDetectorQuery = (detectorId: string) => {
+  return {
+    size: 3,
+    query: {
+      bool: {
+        filter: [
+          {
+            term: {
+              detector_id: detectorId,
+            },
+          },
+          {
+            term: {
+              is_latest: 'true',
+            },
+          },
+          {
+            terms: {
+              task_type: [
+                'REALTIME_HC_DETECTOR',
+                'REALTIME_SINGLE_ENTITY',
```

Comment:
Plan to change `REALTIME_SINGLE_ENTITY` to `REALTIME_SINGLE_FLOW` to avoid confusion with HC entity.

Reply:
I know backend already uses 'single entity' in many places to describe these kinds of detectors. Can you explain the confusion?

Reply:
I remember Kaituo mentioned some word changes, can't recall the details. @kaituo do you have any concern about the single entity name ? If it's ok, I'm fine to keep `_SINGLE_ENTITY`